var/spack/repos/builtin/packages/charmpp/package.py

```diff
@@ -189,7 +189,7 @@ def charmarch(self):
             ("linux", "arm", "netlrts"): "netlrts-linux-arm7",
             ("linux", "arm", "multicore"): "multicore-arm7",
```

Comment:
```suggestion
            ("linux", "arm", "multicore"): "multicore-linux-arm7",
```
Untested, but I assume the same fix is needed here...

Reply:
Yes, I think you're right. Could you please add this to the PR?

Reply:
Done. Thanks for the review @matthiasdiener.